rsmtool/rsmexplain.py

```diff
@@ -476,7 +483,8 @@ def generate_report(explanation, output_dir, ids, configuration, logger=None):
     notebooks_path = Path(__file__).parent / "notebooks"
     notebooks_path = notebooks_path.resolve()
     explanation_notebooks_path = notebooks_path / "explanations"
-    if configuration["show_auto_cohorts"]:
+    # auto cohort plots will be displayed with more than one example selected
+    if configuration["show_auto_cohorts"] and len(explanation.values) > 1:
```

Comment:
Why not use `not has_single_example` here?

Reply:
Could you tell me how to do that? `has_single_example` is defined in the `shap_plots.ipynb`, not sure how to import it here.

Reply:
Right, instead of defining it in the notebook, define it here and then pass it to the report just like we do the other stuff?